### PR TITLE
refactor(color-picker): tailwindify

### PR DESCRIPTION
**Pull request by julio8a**
#Supports #1500

### CODE REVIEW

**Review by paulcpederson — APPROVED** [review]

**Review by paulcpederson — CHANGES_REQUESTED** [review]
the border for the top section is screwed up
Screen Shot 2021-03-19 at 10 00 54 AM

**Comment by julio8a** [comment]
I know why! I think for a few PRs I forgot to add `border-0` first. dang it

File: src/components/calcite-color-picker/calcite-color-picker.scss [hunk 1]
```diff
@@ -14,15 +14,15 @@ $gap--large: 12px;
   }
 
   .channels {
-    flex-direction: column;
+    @apply flex-col;
   }
 
   .channel {
+    @apply w-full mb-1;
     width: 100%;
```
Comment (jrowlingson) [inline src/components/calcite-color-picker/calcite-color-picker.scss:22]: kill

File: src/components/calcite-color-picker/calcite-color-picker.scss [hunk 2]
```diff
@@ -84,95 +82,94 @@ $gap--large: 12px;
...
-  background-color: var(--calcite-ui-foreground-1);
+  @apply inline-block
+  border
+  border-solid
+  border-color-1
+  bg-foreground-1;
 }
 
 .color-field-and-slider {
   margin-bottom: -16px;
```
Comment (jrowlingson) [inline src/components/calcite-color-picker/calcite-color-picker.scss:98]: `-mb-4`

File: src/components/calcite-color-picker/calcite-color-picker.scss [hunk 3]
```diff
@@ -84,95 +82,94 @@ $gap--large: 12px;
...
-    cursor: pointer;
+    @apply cursor-pointer;
   }
 }
 
 .control-section {
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
+  @apply flex flex-row flex-wrap;
```
Comment (jrowlingson) [inline src/components/calcite-color-picker/calcite-color-picker.scss:106]: `flex-row` needed?

File: src/components/calcite-color-picker/calcite-color-picker.scss [hunk 4]
```diff
@@ -84,95 +82,94 @@ $gap--large: 12px;
...
-
-  cursor: pointer;
+  @apply focus-base cursor-pointer;
 
   &:focus {
-    @include focus-style-outset();
+    @apply focus-outset;
   }
 
   &:hover {
```
Comment (jrowlingson) [inline src/components/calcite-color-picker/calcite-color-picker.scss:175]: Missed?

**Comment by jrowlingson** [comment]
Can we kill the vars at the top of the file? There's also a few spots with hard coded `px` widths - could we alter these to become even int rems?

File: src/components/calcite-color-picker/calcite-color-picker.scss (hunk 4 above)
Comment (julio8a) [reply src/components/calcite-color-picker/calcite-color-picker.scss:175]: actually, I didn't know how to approach this.

**Comment by julio8a** [comment]
vars dead!

**Comment by julio8a** [comment]
border fixed @paulcpederson

File: src/components/calcite-color-picker/calcite-color-picker.scss (hunk 4 above)
Comment (jrowlingson) [reply src/components/calcite-color-picker/calcite-color-picker.scss:184]: Lets add this to the list

**Review by jrowlingson — APPROVED** [review]

**Comment by julio8a** [comment]
closing this pr and submitting this file with input

**Pull request closed without merging**